src/panels/logbook/ha-logbook.ts

```diff
@@ -150,6 +154,13 @@ export class HaLogbook extends LitElement {
     this._throttleGetLogbookEntries();
   }
 
+  protected firstUpdated(changedProps: PropertyValues) {
+    super.firstUpdated(changedProps);
+    this.addEventListener("hass-logbook-live", (ev) =>
```

Comment:
The event listener should also be removed when the element is removed from the DOM

Reply:
16f5a49822972254e00fd60fe0a0dde5a89c2c6b